src/ftw_cli/model.py

```diff
@@ -198,175 +198,12 @@ def test(checkpoint_fn, root_dir, gpu, countries, postprocess, iou_threshold, ou
     print(f"Object level precision: {object_precision:.4f}")
     print(f"Object level recall: {object_recall:.4f}")
 
-    if output_fn is not None:
-        if not os.path.exists(output_fn):
-            with open(output_fn, "w") as f:
+    if output is not None:
+        if not os.path.exists(output):
+            with open(output, "w") as f:
                 f.write("train_checkpoint,test_countries,pixel_level_iou,pixel_level_precision,pixel_level_recall,object_level_precision,object_level_recall\n")
-        with open(output_fn, "a") as f:
-            f.write(f"{checkpoint_fn},{countries},{pixel_level_iou},{pixel_level_precision},{pixel_level_recall},{object_precision},{object_recall}\n")
-
-
-# Define the 'inference' command under 'model'
-@click.command(help="Run inference on a satellite image")
-@click.option('--input_fn', type=click.Path(exists=True), required=True, help="Input raster file (Sentinel-2 L2A stack).")
-@click.option('--model_fn', type=click.Path(exists=True), required=True, help="Path to the model checkpoint.")
-@click.option('--output_fn', type=str, required=True, help="Output filename.")
-@click.option('--resize_factor', type=int, default=2, help="Resize factor to use for inference.")
-@click.option('--gpu', type=int, help="GPU ID to use. If not provided, CPU will be used by default.")
-@click.option('--patch_size', type=int, default=1024, help="Size of patch to use for inference.")
-@click.option('--batch_size', type=int, default=2, help="Batch size.")
-@click.option('--padding', type=int, default=64, help="Pixels to discard from each side of the patch.")
-@click.option('--overwrite', is_flag=True, help="Overwrite outputs if they exist.")
-@click.option('--polygonize', is_flag=True, help="Additionally polygonize the output (a GPKG file identical to `output_fn` will be created).")
-@click.option('--simplify', type=float, default=None, help="Simplification factor to use when polygonizing.")
-@click.option('--mps_mode', is_flag=True, help="Run inference in MPS mode (Apple GPUs).")
-def inference(input_fn, model_fn, output_fn, resize_factor, gpu, patch_size, batch_size, padding, overwrite, polygonize, simplify, mps_mode):
```

Comment:
The whole function seems unused as it's now in inference.py?!

Reply:
Oh yeah - sorry, I forgot to remove it when I moved it over.